client/me/pending-payments/pending-list-item.jsx

```diff
@@ -3,33 +3,94 @@
  * External dependencies
  */
 import React from 'react';
-import { localize } from 'i18n-calypso';
+import { localize, moment } from 'i18n-calypso';
 import Gridicon from 'gridicons';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
 
 /**
  * Internal dependencies
  */
 import Card from 'components/card';
 import Button from 'components/button';
+import { getSite, getSiteTitle, getSiteDomain } from 'state/sites/selectors';
+import PurchaseSiteHeader from '../purchases/purchases-site/header';
+import { purchaseType as getPurchaseType, getName } from 'lib/purchases';
+import { paymentMethodName } from 'lib/cart-values';
+import formatCurrency from 'lib/format-currency';
+
+export function PendingListItem( {
+	translate,
+	paymentType,
+	totalCost,
+	currency,
+	siteId,
+	siteTitle,
+	siteDomain,
+	dateCreated,
+	products,
+} ) {
+	let productName = '';
+	let purchaseType = '';
+
+	if ( products.length > 1 ) {
+		productName = 'Multiple Items';
+		purchaseType = 'Various';
+	} else if ( products[ 0 ] ) {
+		productName = getName( products[ 0 ] );
+		purchaseType = getPurchaseType( products[ 0 ] );
+	} else {
```

Comment:
If the previous comment is invalid, then could we remove this final `else` since we've already assigned empty string values in the initial declaration?